kolibri/plugins/management/assets/src/views/user-page/user-create-modal.vue

```diff
@@ -54,7 +54,7 @@
 
       <!-- Button Options at footer of modal -->
       <section class="footer">
-        <k-button :text="$tr('createAccount')" :primary="true" :loading="loading"/>
+        <k-button :text="$tr('createAccount')" :primary="true" :loading="loading" type="submit"/>
```

Comment:
k-button's `type` prop:
https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/views/k-button/index.vue#L53

`loading` is a leftover prop. :disappointed: 

I added the following to the forms doc:

Submission Handing
A form should have the following markup.
`<form @submit.prevent="submitFormMethod">`
Then pass in the type=”submit” prop to the submit button.
Do not attach @click event handlers to buttons or a double submission might occur.